### PR TITLE
feat: add support for fe configured translations

### PR DESCRIPTION
**Pull request by mbritense**

### CODE REVIEW

File: projects/valtimo/components/src/lib/components/confirmation-modal/confirmation-modal.component.html [hunk 1]
```diff
@@ -58,6 +59,21 @@ <h3 cdsModalHeaderHeading>
             obs.cancelTranslation
         }}
       </button>
+      <div *ngIf="spacerAfterCancelButton" class="modal-spacer"></div>
```
Comment (teodor-ritense) [inline projects/valtimo/components/src/lib/components/confirmation-modal/confirmation-modal.component.html:62]: Is "spacer" right or is it a typo?

File: projects/valtimo/components/src/lib/components/confirmation-modal/confirmation-modal.component.html [hunk 2]
```diff
@@ -58,6 +59,21 @@ <h3 cdsModalHeaderHeading>
...
+      <button
+        *ngIf="showOptionalButton"
+        [attr.modal-primary-focus]="true"
+        [cdsButton]="optionalButtonType"
+        (click)="optional()"
```
Comment (teodor-ritense) [inline projects/valtimo/components/src/lib/components/confirmation-modal/confirmation-modal.component.html]: Could you change this to "onOptionalClick"?

File: projects/valtimo/config/src/lib/translation/custom-multi-translate-http-loader.ts [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+      else path = `${resource}${lang}.json`;
+      return new HttpClient(this._handler).get(path).pipe(
+        catchError(res => {
+          if (!resource.optional) {
+            console.group();
+            console.error('Something went wrong for the following translation file:', path);
+            console.error(res);
+            console.groupEnd();
+          }
+          return of({});
```
Comment (teodor-ritense) [inline projects/valtimo/config/src/lib/translation/custom-multi-translate-http-loader.ts:54]: Can this return an object with the error message?

Comment (mbritense) [reply projects/valtimo/config/src/lib/translation/custom-multi-translate-http-loader.ts:54]: This was originally not my code but it's from the multi translate loader guy, so I'm hesitant to change it

**Review by teodor-ritense — DISMISSED** [review]

**Review by teodor-ritense — APPROVED** [review]

**Pull request merged**